CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### 4.0.1 (Next)
 
+* [#215](https://github.com/alexa-js/alexa-app/pull/215): Fixed syntax of session.details to match with session as passed by amazon. - [@danielstieber](https://github.com/danielstieber).
```

Comment:
Maybe "Fixed syntax of `session.details` to match with session object in the request"? Also remove the period inside (after amazon.).